test/manifests/kustomization.yaml

```diff
@@ -4,4 +4,4 @@ kind: Kustomization
 # Actual image overrides will be added in test scripts.
 images: []
 resources:
-- ../../manifests/kustomize/env/dev
+- ../../manifests/kustomize/env/dev
```

Comment:
nit: redundant new line. Perhaps due to auto lint.

Reply:
There is no new line added here.

Reply:
the change was actually deleting newline at end of file.
recommend reverting unrelated changes